src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md

```diff
@@ -67,20 +67,20 @@ You can comment on the proposal explainers by filing issues against each reposit
 Establish the range of web activity across which the user's browser can let websites treat a person as having a single identity. Identify the ways in which information can move across identity boundaries without compromising that separation.
 * [Privacy Budget](https://github.com/bslassey/privacy-budget)<br>
 Limit the total amount of potentially identifiable data that sites can access. Update APIs to reduce the amount of potentially identifiable data revealed. Make access to potentially identifiable data measurable.
-* [Willful IP Blindness](https://github.com/bslassey/ip-blindness)<br>
-Enable sites to 'blind' themselves to IP addresses so they can avoid consuming privacy budget.
+* [Gnatcatcher](https://github.com/bslassey/ip-blindness)<br>
+Limit the ability to identify individual users by accessing their IP address. 
 * [Trust Token API](https://github.com/dvorak42/trust-token-api)<br>
 Enable an origin that trusts a user to issue them with cryptographic tokens which are stored by the user's browser so they can be used in other contexts to evaluate the user's authenticity.
 * [First-Party Sets](https://github.com/krgovind/first-party-sets/)<br>
 Allow related domain names owned by the same entity to declare themselves as belonging to the same first party.
 * [Aggregated Reporting](https://github.com/csharrison/aggregate-reporting-api)<br>
 Provide privacy preserving mechanisms to support a variety of use cases such as view-through-conversion, brand, lift, and reach measurement.
-* [Click Through Conversion Measurement Event-Level](https://github.com/csharrison/conversion-measurement-api)<br>
-Provide privacy preserving [click-through-conversion](#glossary-ctc) measurement.
-* [Federated Learning of Cohorts](https://github.com/jkarlin/floc)<br>
+* [Attribution Reporting](https://github.com/csharrison/conversion-measurement-api)<br>
```

Comment:
Minor issue because that links redirects to this one anyway, but:
```suggestion
* [Attribution Reporting](https://github.com/WICG/conversion-measurement-api)<br>
```